itwin-platform-access/imodels-access-frontend/src/FrontendIModelsAccess.ts

```diff
@@ -84,4 +98,30 @@ export class FrontendIModelsAccess implements FrontendHubAccess {
       return PlatformToClientAdapter.toAuthorization(token);
     };
   }
+
+  private async _getChangesetFromLatestNamedVersion(arg: IModelIdArg): Promise<ChangesetIndexAndId> {
+    const getNamedVersionListParams: GetNamedVersionListParams = {
```

Comment:
could be simplified to:
`const getNamedVersionListParams: GetNamedVersionListParams = this.getIModelScopedOperationParams(arg);`